<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsOfTheProgramWindow</class>
 <widget class="QMainWindow" name="SettingsOfTheProgramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>936</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>936</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="settingsOfTheProgramCentralWidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>936</width>
      <height>618</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgb(255, 255, 255);
	border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="leftMenuFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>618</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: rgb(245, 245, 245);
	border: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="accountSettingFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-radius: 0px;
	border-bottom: 2px solid rgb(240, 240, 240);
	border-top-left-radius: 15px;
}

QFrame::hover {
	background-color: rgb(248, 248, 248);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="iconOfAccountSettings">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: none;
	border: none;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/memberIcon_B.png</normaloff>:/memberIcon_B.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="titleOfAccountSettings">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>28</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Account</string>
       </property>
      </widget>
      <widget class="QLabel" name="descriptionOfAccountSettings">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>9</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Change your personal information.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>696</width>
       <height>618</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QStackedWidget {
	background-color: rgb(255, 255, 255);
	border-top-right-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="accountSettingsPage">
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: rgb(255, 255, 255);
	border-top-right-radius: 15px;
	border-bottom-right-radius: 15px;
}</string>
      </property>
      <widget class="QLabel" name="accountAvatar">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>211</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: none;
	border: none;
	border-radius: 0px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../icons/icons.qrc">:/maleMemberIcon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="changeTheAvatarButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>230</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(83, 200, 240);
	border: none;
	border-radius: 25px;
}

QPushButton:hover {
	background-color: rgb(91, 202, 240);
}

QPushButton:pressed {
	background-color: rgb(75, 197, 239);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/editSomethingIconBorder_W.png</normaloff>:/editSomethingIconBorder_W.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="Line" name="separated_line">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>601</width>
         <height>2</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 245, 245);</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="blankForEmailAddress">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>13</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(245, 245, 245);
	border-radius: 7px;
	padding: 12px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>E-mail address</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="blankForUsername">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>350</y>
         <width>291</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>13</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(245, 245, 245);
	border-radius: 7px;
	padding: 12px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="blankForPassword">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>450</y>
         <width>601</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>13</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(245, 245, 245);
	border-radius: 7px;
	padding: 12px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="emailLabel">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>327</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
       </property>
       <property name="text">
        <string>E-mail address</string>
       </property>
      </widget>
      <widget class="QLabel" name="usernameLabel">
       <property name="geometry">
        <rect>
         <x>352</x>
         <y>327</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="passwordLabel">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>427</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: none;
	border: none;
}</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>540</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(214, 214, 214);
	border-radius: 7px;
	color: rgb(255, 255, 255);
	border: none;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
}

QPushButton:pressed {
	background-color: rgb(202, 202, 202);
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveTheSettingsButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>540</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Montserrat</family>
         <pointsize>13</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(56, 236, 113);
	border-radius: 7px;
	color: rgb(255, 255, 255);
	border: none;
}

QPushButton:hover {
	background-color: rgb(69, 241, 124);
}

QPushButton:pressed {
	background-color: rgb(48, 221, 103);
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
